scss/_patterns_grid.scss

```diff
@@ -137,7 +137,7 @@
   @media (min-width: $threshold-6-12-col) {
     @for $i from $grid-columns through 1 {
       // set col-* to span respective number of columns on desktop
-      .#{$grid-desktop-col-prefix}#{$i} {
+      .#{$grid-big-col-prefix}#{$i} {
```

Comment:
We usually use "large" for screen viewports (yes, "large" and "big" are two words for _almost_ the same thing and English is stupid)

Reply:
large sounds a lot better, thanks. Updated here and in a few other places.